.travis.yml

```diff
@@ -1,8 +1,8 @@
 language: python
 python:
-  - "2.7"
```

Comment:
Can you keep 2.7 version as well? and manage the version in the code with `if/else`.

Reply:
I never use Python 2.7 and I don't have the setup for. It's possible to re-up travis? It can help.

Reply:
Other People use it, Add 2.7 python version and please update Travis to pass all tests.

Reply:
Sorry, but I don't have time to support old version. This pull request it's juste the modification I have implemented for it to work. Because the code in master it's not usable (many errors on Windows). You can close the PR.